test/hotspot/jtreg/compiler/vectorization/TestNumberOfContinuousZeros.java

```diff
@@ -40,6 +42,7 @@
 import jdk.test.lib.Asserts;
 
 public class TestNumberOfContinuousZeros {
+    private static final int[] SPECIAL = { 0x01FFFFFF, 0x03FFFFFE, 0x07FFFFFC, 0x0FFFFFF8, 0x1FFFFFF0, 0x3FFFFFE0 };
```

Comment:
Can you also check for 0xFFFFFFFF, even though we have special handling for -ve signed numbers not affected by this patch.